docs/examples/gnmi-clients/gnmic/index.md

```diff
@@ -61,18 +49,881 @@ gnmic -a 127.0.0.1:6030 -u admin -p admin --insecure capabilities  \
+```
+
+<details><summary> Reveal output</summary>
+<p>
+
+```javascript
+{
+  "source": "10.83.13.214:6030",
+  "subscription-name": "default-1626303017",
+  "timestamp": 1626302991598465536,
+  "time": "2021-07-14T23:49:51.598465536+01:00",
+  "updates": [
+    {
+      "Path": "network-instances/network-instance[name=Tenant_A_WEB_Zone]/protocols/protocol[identifier=BGP][name=BGP]/bgp/neighbors/neighbor[neighbor-address=10.255.251.21]/state/last-established",
+      "values": {
+        "network-instances/network-instance/protocols/protocol/bgp/neighbors/neighbor/state/last-established": 1626302991563603200
+      }
+    }
+  ]
+}
+```
+
+</p>
+</details>
+
+#### Subscribe with stream mode sample and interval
+
+```shell
+gnmic -a 10.83.13.214:6030 -u cvpadmin -p arastra --insecure --gzip --sample-interval 5s --stream-mode sample subscribe --path  \
+  '/network-instances/network-instance[name=Tenant_A_WEB_Zone]/protocols/protocol[identifier=BGP][name=BGP]/bgp/neighbors/neighbor[neighbor-address=10.255.251.21]/afi-safis/afi-safi[afi-safi-name=openconfig-bgp-types:IPV4_UNICAST]/state/prefixes/received'
+```
+
+<details><summary> Reveal output</summary>
+<p>
+
+```javascript
+{
+  "source": "10.83.13.214:6030",
+  "subscription-name": "default-1626303393",
+  "timestamp": 1626303158135414182,
+  "time": "2021-07-14T23:52:38.135414182+01:00",
+  "updates": [
+    {
+      "Path": "network-instances/network-instance[name=Tenant_A_WEB_Zone]/protocols/protocol[identifier=BGP][name=BGP]/bgp/neighbors/neighbor[neighbor-address=10.255.251.21]/afi-safis/afi-safi[afi-safi-name=IPV4_UNICAST]/state/prefixes/received",
+      "values": {
+        "network-instances/network-instance/protocols/protocol/bgp/neighbors/neighbor/afi-safis/afi-safi/state/prefixes/received": 5
+      }
+    }
+  ]
+}
+{
+  "source": "10.83.13.214:6030",
+  "subscription-name": "default-1626303393",
+  "timestamp": 1626303158135414182,
+  "time": "2021-07-14T23:52:38.135414182+01:00",
+  "updates": [
+    {
+      "Path": "network-instances/network-instance[name=Tenant_A_WEB_Zone]/protocols/protocol[identifier=BGP][name=BGP]/bgp/neighbors/neighbor[neighbor-address=10.255.251.21]/afi-safis/afi-safi[afi-safi-name=IPV4_UNICAST]/state/prefixes/received",
+      "values": {
+        "network-instances/network-instance/protocols/protocol/bgp/neighbors/neighbor/afi-safis/afi-safi/state/prefixes/received": 5
+      }
+    }
+  ]
+}
+```
+
+</p>
+</details>
+
+## gNMI SET RPC Examples
+
+### OpenConfig paths
+
+#### Configure BGP neighbor address and peer AS
+
+```shell
+gnmic -a 10.83.13.214:6030 -u cvpadmin -p arastra --insecure --gzip set \
+--update-path '/network-instances/network-instance[name=default]/protocols/protocol[name=BGP]/bgp/neighbors/neighbor[neighbor-address=10.10.100.43]' \
+--update-file value.json
+```
+
+`value.json`:
+
+`{"config": {"neighbor-address":"10.10.100.43", "peer-as": 123}}`
```

Comment:
this might render a little better if it was split into 2 parts
- turn file name into bolded text?
- triple-backtick the file contents, this seems to break oddly when rendered if the window is narrow.